graph/appender/istio_details.go

```diff
@@ -96,12 +96,13 @@ func applymTLS(n *tree.ServiceNode, namespaceName string, istioDetails *kubernet
 		return
 	}
 
-	serviceName := strings.Split(n.Name, ".")[0]
+	nodeService := strings.Split(n.Name, ".")[0]
```

Comment:
Why split twice ? I would do it in one time, and check that the len() is correct.
Perhaps serviceName is tested outside this method, but it should add a validation, and a single op for Split() IMO.